diff -urN integrit-3.01/Makefile.in integrit-3.01.dzikus/Makefile.in
--- integrit-3.01/Makefile.in	Thu May 16 06:27:47 2002
+++ integrit-3.01.dzikus/Makefile.in	Mon Aug 12 15:42:37 2002
@@ -130,11 +130,11 @@
 	  echo installing auxiliary programs; \
 	  cd utils && $(MAKE) $@; \
 	fi
-	@if test ! -d $(SBINDIR); then \
-	  echo creating directory $(SBINDIR); \
-	  $(INSTALL) -d -m 755 $(SBINDIR); \
+	@if test ! -d $(DESTDIR)$(SBINDIR); then \
+	  echo creating directory $(DESTDIR)$(SBINDIR); \
+	  $(INSTALL) -d -m 755 $(DESTDIR)$(SBINDIR); \
 	fi
-	$(INSTALL) $(STRIP) -m 755 $(PROG) $(SBINDIR)/$(PROG)
+	$(INSTALL) $(STRIP) -m 755 $(PROG) $(DESTDIR)$(SBINDIR)/$(PROG)
 	@echo
 	@echo 'It is recommended that the binary be copied to a secure location and'
 	@echo "  re-copied to $(SBINDIR) at runtime or run directly"
diff -urN integrit-3.01/doc/Makefile.in integrit-3.01.dzikus/doc/Makefile.in
--- integrit-3.01/doc/Makefile.in	Tue Jul 16 01:57:57 2002
+++ integrit-3.01.dzikus/doc/Makefile.in	Wed Aug 14 02:16:36 2002
@@ -33,13 +33,13 @@
 
 install-man :
 	@for n in 1; do \
-	  if test ! -d $(MANDIR)/man$$n; then \
-	    echo creating directory $(MANDIR)/man$$n; \
-	    $(INSTALL) -d $(MANDIR)/man$$n; \
+	  if test ! -d $(DESTDIR)$(MANDIR)/man$$n; then \
+	    echo creating directory $(DESTDIR)$(MANDIR)/man$$n; \
+	    $(INSTALL) -d $(DESTDIR)$(MANDIR)/man$$n; \
 	  fi; \
 	  for f in *.$$n; do \
-	    echo installing manpage $$f in $(MANDIR)/man$$n; \
-	    $(INSTALL) -m 644 $$f $(MANDIR)/man$$n/$$f; \
+	    echo installing manpage $$f in $(DESTDIR)$(MANDIR)/man$$n; \
+	    $(INSTALL) -m 644 $$f $(DESTDIR)$(MANDIR)/man$$n/$$f; \
 	  done; \
 	done
 
@@ -53,35 +53,15 @@
 # so we accomodate at the price of some complexity.
 # 
 install-info : 
-	@if test ! -d $(infodir); then \
-	  echo creating directory $(infodir); \
-	  $(INSTALL) -d $(infodir); \
+	@if test ! -d $(DESTDIR)$(infodir); then \
+	  echo creating directory $(DESTDIR)$(infodir); \
+	  $(INSTALL) -d $(DESTDIR)$(infodir); \
 	fi
 	@for f in $(INFO_FILES); do \
-	  echo installing $$f in $(infodir); \
-	  $(INSTALL) -m 644 $(srcdir)/$$f $(infodir); \
+	  echo installing $$f in $(DESTDIR)$(infodir); \
+	  $(INSTALL) -m 644 $(srcdir)/$$f $(DESTDIR)$(infodir); \
 	done
-	@:; iistatus=0; \
-	if ! $(INSTALL_INFO) --version > /dev/null 2>&1; then \
-	  iistatus=1; \
-	elif test ! -z "`$(INSTALL_INFO) --version 2>&1 | grep texinfo`"; then \
-	  (set -xe; \
-	    $(INSTALL_INFO) --dir-file=${infodir}/dir \
-	                    --info-file=${infodir}/integrit.info < /dev/null \
-	  ) || iistatus=1; \
-	else \
-	  echo " ***                                               *** "; \
-	  echo " *** non-texinfo version of install-info detected. *** "; \
-	  echo " *** see INSTALL for more information              *** "; \
-	  echo " ***                                               *** "; \
-	fi; \
-	if [ $$iistatus != 0 ]; then \
-	  echo "Warning: $(INSTALL_INFO) did not run successfully."; \
-	  echo "         to complete the installation of the documenation,"; \
-	  echo "         please read INSTALL."; \
-	  echo "(pausing 3 seconds)"; \
-	  sleep 3; \
-	fi
+	@:; iistatus=0
 
 distready : info
 
diff -urN integrit-3.01/utils/Makefile.in integrit-3.01.dzikus/utils/Makefile.in
--- integrit-3.01/utils/Makefile.in	Mon Feb  4 05:28:30 2002
+++ integrit-3.01.dzikus/utils/Makefile.in	Mon Aug 12 15:42:57 2002
@@ -100,13 +100,13 @@
 	$(RM) -f Makefile
 
 install : $(PROGS)
-	@if test ! -d $(SBINDIR); then \
-	  echo creating directory $(SBINDIR); \
-	  $(INSTALL) -d -m 755 $(SBINDIR); \
+	@if test ! -d $(DESTDIR)$(SBINDIR); then \
+	  echo creating directory $(DESTDIR)$(SBINDIR); \
+	  $(INSTALL) -d -m 755 $(DESTDIR)$(SBINDIR); \
 	fi
-	$(INSTALL) $(STRIP) -m 755 i-viewdb $(SBINDIR)/i-viewdb
-	@if test ! -d $(BINDIR); then \
-	  echo creating directory $(BINDIR); \
-	  $(INSTALL) -d -m 755 $(BINDIR); \
+	$(INSTALL) $(STRIP) -m 755 i-viewdb $(DESTDIR)$(SBINDIR)/i-viewdb
+	@if test ! -d $(DESTDIR)$(BINDIR); then \
+	  echo creating directory $(DESTDIR)$(BINDIR); \
+	  $(INSTALL) -d -m 755 $(DESTDIR)$(BINDIR); \
 	fi
-	$(INSTALL) $(STRIP) -m 755 i-ls $(BINDIR)/i-ls
+	$(INSTALL) $(STRIP) -m 755 i-ls $(DESTDIR)$(BINDIR)/i-ls
